src/components/pastAnalysis/seriesTable.tsx

```diff
@@ -26,45 +28,66 @@ export const isLargestNumber = (num: number | null | undefined, numArray: Map<st
   }
 }
 
-const SeriesTable: React.FC<SeriesTableProps> = ({ studyInstance, isProcessing }) => {
+const SeriesTable: React.FC<SeriesTableProps> = ({ studyInstance, classifications, isProcessing }) => {
   const history = useHistory();
   const { dispatch } = useContext(AppContext);
   const { series: analysisList } = studyInstance;
+
   let titles = [
     { title: (<span><br />Preview<span className='classificationText'>&nbsp;</span></span>) },
     { title: (<span><br />Image<span className='classificationText'>&nbsp;</span></span>) }
   ];
 
-  analysisList[0]?.classifications.forEach((value: number, key: string) => {
-    titles.push({ title: (<span><br /><span className='classificationText'>{key}</span></span>) }); // Adding the column titles for each analysis
-  });
+  if (classifications.length) {
+    classifications.forEach((value: string) => {
+      titles.push({ title: (<span><br /><span className='classificationText'>{value}</span></span>) }); // Adding the column titles for each analysis
+    });
+  } else {
+    titles.push({ title: (<span><br /><span className='classificationText'>{"Results"}</span></span>) }); // Adding column for studies containing only failed analyses
+  }
 
   titles.push({ title: (<span className='classificationText'><span>Geographic<br />Severity</span></span>) },
-              { title: (<span className='classificationText'><span>Opacity<br />Extent</span></span>) },
-              { title: (<span></span>) });
+    { title: (<span className='classificationText'><span>Opacity<br />Extent</span></span>) },
+    { title: (<span></span>) });
```

Comment:
Remove unnecessary `span` tag.

```suggestion
    { title: (<></>) });
```